mappings/net/minecraft/util/PacketByteBuf.mapping

```diff
@@ -160,6 +160,8 @@ CLASS net/minecraft/class_1435 net/minecraft/util/PacketByteBuf
 	METHOD setShort (II)Lio/netty/buffer/ByteBuf;
 		ARG 1 i
 		ARG 2 j
-	METHOD writeBytes (Lio/netty/buffer/ByteBuf;)Lio/netty/buffer/ByteBuf;
```

Comment:
its a byte buf. not a byte. the correct name is writeBytes

Reply:
i
did
not
do
that

packetbytebuf
has
all
methods
deobf

Reply:
you cant say you didnt do that. its literately a change in the file. from YOUR pr

Reply:
It doesn’t matter
All methods are deobfuscated